rflx/generator/allocator.py

```diff
@@ -233,19 +238,25 @@ def _create_global_allocated_pred(self, slots: List[NumberedSlotInfo]) -> UnitPa
                             for slot in slots
                             if not slot.global_
                         ],
-                    ),
+                    )
+                    if slots
+                    else TRUE,
```

Comment:
That shouldn't be necessary, as `And` is already interpreted as `True` if there are no terms.

https://github.com/Componolit/RecordFlux/blob/f6a5c4e3736e02874866917e7e3fd48344fdcd29/rflx/ada.py#L140-L143.

Reply:
fixed